src/endpoints/rooms/createRoom.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+var mongoose = require("mongoose");
+
+var hash = require("../../auth/hash");
+var helpers = require("../helpers");
+var Token = require("../../auth/token");
+var Room = require("../../models/room");
+var User = require("../../models/user");
+
+var URL = "/api/users/join";
+
+/**
+ * Endpoint handler for createForRoom endpoint.
+ */
+function handle(req, res) {
+	if (!helpers.request.requireBodyFields(req, res, ["name"]) {

```

Comment:
I think you're missing one ')' after the `["name"])` and before the `{ return }`